src/generateSdk/sdkGenerator/JsSdkGenerator.ts

```diff
@@ -29,18 +29,75 @@ const template = `/**
 
 import { Remote } from "./remote.js"
 
+{{#hasGnzContext}}
```

Comment:
this would be added in every class that uses GnzContext, right? wouldn't it be better to add it to the `remote` file to avoid duplicated code? worst case the code won't be used.